lib/tasks/notification.rake

```diff
@@ -0,0 +1,12 @@
+namespace :notification do
+  desc 'TODO'
+  task send_digest: :environment do
+    completed = Item.completed.for_today.count
+    remaining = Item.incompleted.count
+
+    statistic = Hash(completed: completed, remaining: remaining)
+
+    daily_statistic = DailyStatisticNotifier.new statistic
+    Notifier.statistic(daily_statistic).deliver
+  end
```

Comment:
@vlasiak, as you see, this code is duplicated with the temporary code from [the action](https://github.com/vlasiak/rails_tada/pull/16/files#diff-2654c8b19e2ec4760155e69b76ec711aR29). This means that there is no way currently to lunch (to call) the digest in a few lines of code. That is a tip about lack of some abstraction (class).